enterprise/tools/check_metrics/check_metrics.go

```diff
@@ -95,7 +103,21 @@ func main() {
 					}
 
 					if unhealthyCount >= maxUnhealthyCount {
-						return errors.New(fmt.Sprintf("%s metrics unhealthy.", metric.Name))
+						err = func() error {
```

Comment:
why not just do this stuff inline instead of wrapping in a function?

Reply:
Fixed